test/state/state.cpp

```diff
@@ -107,6 +149,18 @@ Account& State::get_or_insert(const address& addr, Account account)
     return insert(addr, std::move(account));
 }
 
+bytes_view State::get_code(const address& addr)
```

Comment:
Can be `const` method?

Reply:
No. We potentially load an account and/or its code from initial state.